server/src/main/java/meghanada/completion/JavaCompletion.java

```diff
@@ -237,7 +235,6 @@ private static CompletionMatcher createClassCompletionMatcher(
     final CompletionMatcher classMatcher = getClassCompletionMatcher(prefix);
     // add this member
     completionThisMembers(typeScope, prefix, result, matcher);
-    completionThisMembers(prefix, result, fqcn);
```

Comment:
Why did you delete it ?

Reply:
Dear,

I found that when completion in current class, for example,  take the follow code for example,

```
class A {
   public void aMethod() {}

  public void bMethod() {
    this. //<-- Cursor here
  }
}
```

aMethod will popup for two times, and after I deleted the above line, it only appears for one time.

Reply:
I see. OK.